front/activity_bar.html

```diff
@@ -0,0 +1,36 @@
+<html>
+    <head>
+        <script src="https://unpkg.com/vue@next"></script>
+        <link rel="stylesheet" href="css/global.css">
+        <link rel="stylesheet" href="css/activity_bar.css">
+    </head>
+    <body>
+        <div id="app">
+            <activity-bar v-bind:current-room="currentRoom.title" v-bind:is-writing="isWriting"></activity-bar>
```

Comment:
Great but we should have a `writting-start` and `writting-end` event on the `message-editor` and also handle it on the server side. This won't be possible in the first version, might be possible when we add websockets.

Reply:
yea i wasn't really sure how to implement it but since this PR is for activity-bar only i didn't really look at the actuall implementation, but i could make another one if needed

Reply:
We will see that in version 2

Reply:
Does this "isWriting" state means that someone is writting ? or that the current user is writting ?

Reply:
Issue here : #119